webapp/views/wpt-app.js

```diff
@@ -159,6 +165,11 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
         type: String,
         computed: '_computePath(subroute.path)',
       },
+      metadata: Array,
```

Comment:
Perhaps this, along with `displayedMetadata` and related methods, should go to `wpt-metadata`. And then `<wpt-metadata>` would take only `path` as from `wpt-app`. @lukebjerring might have more experience regarding the tradeoff here.